src/lib/init.js

```diff
@@ -49,10 +49,25 @@ export function init(configUpdates) {
 
 				// set cookies on digitrust domain after consent submitted
 				const { addEventListener, getVendorConsents } = cmp.commands;
+				if (config.digitrust.pixel) {
+					addEventListener('consentStringUpdated', digitrustPixel);
```

Comment:
The string "consentStringUpdated" might be a constant.

Reply:
I couldn't find it defined anywhere... not to say that it shouldn't be.